pkg/doc/verifiable/credential.go

```diff
@@ -712,7 +712,7 @@ func loadCredentialSchema(url string, client *http.Client) ([]byte, error) {
 	var gotBody []byte
 	gotBody, err = ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return nil, fmt.Errorf("reading response body failed: %w", err)
+		return nil, fmt.Errorf("credential schema: read response body: %w", err)
```

Comment:
shouldn't use a : in the middle of the message, as it is being used as the wrapping separator.

Reply:
Yeah, I will fix it in the upcoming PRs!